client/my-sites/site-settings/sitemaps.jsx

```diff
@@ -0,0 +1,196 @@
+	};
+
+	renderSitemapLink( sitemapUrl ) {
+		return (
+			<div key={ sitemapUrl }>
+				<ExternalLink href={ sitemapUrl } icon target="_blank">
+					{ sitemapUrl }
+				</ExternalLink>
+			</div>
+		);
+	}
+
+	renderInfoLink( url ) {
+		const { translate } = this.props;
+
+		return (
+			<div className="sitemaps__info-link-container site-settings__info-link-container">
+				<InfoPopover position={ 'left' }>
+					<ExternalLink href={ url } icon target="_blank">
+						{ translate( 'Learn more about Sitemaps.' ) }
+					</ExternalLink>
+				</InfoPopover>
+			</div>
+		);
+	}
+
+	renderSitemapExplanation() {
+		const { translate } = this.props;
+
+		return (
+			<FormSettingExplanation>
+				{ translate( 'Your sitemap is automatically sent to all major search engines for indexing.' ) }
+			</FormSettingExplanation>
+		);
+	}
+
+	renderWpcomSettings() {
+		const { site } = this.props;
+		const sitemapTypes = [
+			'sitemap',
+			'news-sitemap',
+		];
+
+		return (
+			<div>
+				{ this.renderInfoLink( 'https://support.wordpress.com/sitemaps/' ) }
+
+				{ this.renderSitemapExplanation() }
+
+				{
+					sitemapTypes.map( ( sitemapType ) => {
+						const sitemapUrl = site.URL + '/' + sitemapType + '.xml';
+						return this.renderSitemapLink( sitemapUrl );
+					} )
+				}
+			</div>
+		);
+	}
+
+	renderJetpackSettingsContent() {
+		const {
+			activatingSitemapsModule,
+			sitemapsModule,
+			sitemapsModuleActive,
+			translate
+		} = this.props;
+
+		if ( ! activatingSitemapsModule && ! sitemapsModuleActive ) {
+			return;
+		}
+
+		const sitemapTypes = [
+			'sitemap_url',
+			'news_sitemap_url',
+		];
+
+		return (
+			<div className="sitemaps__module-settings site-settings__child-settings">
+				{
+					activatingSitemapsModule &&
+					<FormSettingExplanation>
+						{ translate( 'Generating sitemap…' ) }
+					</FormSettingExplanation>
+				}
+
+				{
+					sitemapsModuleActive &&
+					<div>
+						{ this.renderSitemapExplanation() }
+
+						{
+							sitemapsModule &&
+							sitemapTypes.map( ( sitemapType ) =>
+								sitemapsModule.extra[ sitemapType ] &&
+								this.renderSitemapLink( sitemapsModule.extra[ sitemapType ] )
+							)
+						}
+					</div>
+				}
+			</div>
+		);
+	}
+
+	renderJetpackSettings() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			siteId,
+			translate
+		} = this.props;
+
+		return (
+			<FormFieldset>
+				{ this.renderInfoLink( 'https://jetpack.com/support/sitemaps' ) }
+
+				<JetpackModuleToggle
+					siteId={ siteId }
+					moduleSlug="sitemaps"
+					label={ translate( 'Generate XML sitemaps' ) }
+					disabled={ isRequestingSettings || isSavingSettings }
+				/>
+
+				{ this.renderJetpackSettingsContent() }
+			</FormFieldset>
+		);
+	}
+
+	render() {
+		const {
+			siteId,
+			siteIsJetpack,
+			translate
+		} = this.props;
+
+		return (
+			<div>
+				<QueryJetpackConnection siteId={ siteId } />
+
+				<SectionHeader label={ translate( 'Sitemaps' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Sitemap' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).